ita-j-agronomy.csl

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal" default-locale="en-GB">
+  <info>
+    <title>Italian Journal of Agronomy</title>
+    <id>http://www.zotero.org/styles/ita-j-agronomy</id>
+    <link href="http://www.zotero.org/styles/ita-j-agronomy" rel="self"/>
```

Comment:
can you write the ID, filename and self-link with the full name? italian-journal-of-agronomy